kafka-agent/src/main/java/io/strimzi/kafka/agent/KafkaAgent.java

```diff
@@ -255,15 +259,12 @@ public boolean handle(Request request, Response response, Callback callback) thr
         };
     }
 
-    private SslContextFactory.Server getSSLContextFactory() {
+    private SslContextFactory.Server getSSLContextFactory() throws GeneralSecurityException, IOException {
         SslContextFactory.Server sslContextFactory = new SslContextFactory.Server();
+        sslContextFactory.setTrustStore(KafkaAgentUtils.jksTrustStore(caCertSecret));
 
-        sslContextFactory.setKeyStorePath(sslKeyStorePath);
-        sslContextFactory.setKeyStorePassword(sslKeyStorePassword);
-        sslContextFactory.setKeyManagerPassword(sslKeyStorePassword);
-
-        sslContextFactory.setTrustStorePath(sslTruststorePath);
-        sslContextFactory.setTrustStorePassword(sslTruststorePassword);
+        sslContextFactory.setKeyStore(KafkaAgentUtils.jksKeyStore(nodeCertSecret));
+        sslContextFactory.setKeyStorePassword("changeit");
```

Comment:
what about this "changeit"? We had a random generated password when using the script.

Reply:
I had followed what KafkaAgentClient had, but you are right, we should do the same as the script. I added a method to generate random password with 32 chars like the script did.